aws/websites/metrics.pytorch.org/update_dashboards.py

```diff
@@ -88,8 +88,14 @@ def file_path(name: str) -> Path:
         with open(file["path"], "w") as f:
             f.write(dashboard_on_grafana)
 
+    #delete dashboard use case
+    dashboards_uids = { d["uid"]: d["uid"] for d in dashboards }
```

Comment:
you can use a set comprehension here instead since the value doesn't matter
```suggestion
    dashboards_uids = { d["uid"] for d in dashboards }
```